server/src/controllers/users.js

```diff
@@ -103,6 +103,8 @@ class UserController {
     const userJWT = jwt.sign(
       {
         id: newUser.id,
+        firstName: newUser.firstName,
+        lastName: newUser.lastName,
```

Comment:
I don't think we need this right?

Reply:
Is it okay to add a DB call instead when we get current user for frontend?